### PR TITLE
Fix pagination when search is done by name

### PR DESCRIPTION
**Pull request by LeMeteore**
I was not able to really check/test on my laptop, because no fixtures :( Let me know if something went wrong.
For the description, please see https://github.com/healthsites/healthsites/issues/921

### CODE REVIEW

**Comment by meomancer** [comment]
LGTM

**Comment by dodobas** [comment]
@LeMeteore I can share my development database if that will help, just hop in the gitter chat room and contact me ..

File: django_project/localities/models.py [hunk 1]
```diff
@@ -291,7 +291,8 @@ def repr_dict(self, clean=False):
             pass
 
         # exclusive for open street map
-        if "openstreetmap" in self.upstream_id.lower():
+        # if "openstreetmap" in self.upstream_id.lower():
```
Comment (dodobas) [inline django_project/localities/models.py]: please remove dead code, if it's not used we don't need it

Comment (LeMeteore) [inline django_project/localities/models.py]: Yeah, sure. But in the last commit, there is no dead code :\

**Pull request merged**